nginx/parts/plugins/x-autotools.py

```diff
@@ -0,0 +1,62 @@
+"""This is a modified version of snapcraft's autotools plugin.
+
+It calls nginx' ./auto/configure
```

Comment:
I think that's "nginx'" instead of "nginx's" (possessive apostrophe after 's', 'x', 'z'), but maybe it could be rephrased to make it clearer? Anyway a minor point.